src/login.coffee

```diff
@@ -42,7 +42,8 @@ define [
         $.each @my.popupTypes, (index, type) =>
           @_bindForms type
 
-        $("a.logout").click (e) => @_logOut e
+        $(document).on 'click', 'a.logout', (e) =>
```

Comment:
Much, much better.

Just a heads up. This may conflict with AG's version of jquery-autotagging (which calls `event.stopPropagation`?!). It's [in the works to remove it](https://www.pivotaltracker.com/story/show/81968454).